src/fragments/lib/push-notifications/android/record_notifications/10_handle_notification_received.mdx

```diff
@@ -0,0 +1,104 @@
+<Callout>
+
+These APIs do not need to be called unless you choose to not use our service extension or you want to support multiple push providers.
```

Comment:
I think maybe the wording should be "You do not need to trigger these APIs manually if you are using our library out of the box" (or something similar). It just seems to me that we introduce the service extension for the first time here - which is confusing.